src/Sarif.Converters.UnitTests/ConverterTestsBase.cs

```diff
@@ -13,6 +14,7 @@ public class ConverterTestsBase<T> where T : ToolFileConverterBase, new()
     {
         public SarifLog RunTestCase(string inputData, string expectedResult, bool prettyPrint = true)
         {
+            PrereleaseCompatibilityTransformer.UpdateToCurrentVersion(expectedResult, forceUpdate: false, formatting: Formatting.Indented, out expectedResult);
```

Comment:
>PrereleaseCompatibilityTransformer [](start = 12, length = 34)

Previously, we made manual updates to expected test results flowing through this helper.